Dockerfile

```diff
@@ -28,8 +29,9 @@ RUN apt-get update \
     uuid-runtime \
     tzdata \
     jq \
-  && apt remove --purge -y software-properties-common \
-  && apt autoremove --purge -y \
+  && if [ "$(echo "${version}" | cut -d. -f1)" -ge 18 ]; then apt-get install gpg; fi \
```

Comment:
Does this fail if added to the list above?

Reply:
It needs a 18.04+ test because it's not called `gpg` on 16.04 (it's called `gnupg` on 16.04 and is already installed anyway).

Reply:
```
# grep VERSION_ID /etc/os-release 
VERSION_ID="16.04"
# apt-get install jq gpg
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package gpg is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'gpg' has no installation candidate
```